### ISSUE TITLE
Invalid input "failWithIssues"

### ISSUE THREAD

**Issue by zyriab** [netlify-labs/netlify-plugin-a11y#94]
**Describe the bug**
When running the plugin on Netlify build, I have the following error:
```
Plugin "netlify-plugin-a11y" invalid input "failWithIssues"

Unknown inputs for plugin "netlify-plugin-a11y": "failWithIssues"
Plugin inputs should be one of: "checkPaths", "ignoreDirectories", "resultMode", "debugMode", "testMode"
Check your plugin configuration to be sure that:


the input name is spelled correctly
the input is included in the plugin's available configuration options
the plugin's input requirements have not changed

Plugin inputs
failWithIssues: 'false'

While loading "netlify-plugin-a11y" from netlify.toml
```

I am deploying a simple html/css/js website with barely anything in the files as I am testing the use of various plugins

**To Reproduce**
1. Deploy a website with GitHub with the plugin in a `netlify.toml` file

**Expected behavior**
The option `failWithIssues` should be recognized by the plugin.

**Screenshots**
<img width="960" alt="image" src="https://github.com/netlify-labs/netlify-plugin-a11y/assets/2111910/c40fecba-e6c6-4d04-9ea1-f18c39edebda">

**Desktop (please complete the following information):**
    - OS: MacOS Sonoma 14.2.1
    - Browser Firefox Developer Edition
    - Version 122.0b9 (64bits)

**Additional context**
Here's my full `.toml` file when running this build (sorry for the mess).
The plugin is **4th when starting from the bottom of the file***.
```toml
[build]
    # publish = "public"

[build.environment]
    GO_VERSION = "1.21.x"

[build.processing.html]
    pretty_urls = true

[functions]
    directory = "functions/"

[dev]
    debugMode = true

# [context.dev.environment]

# TODO: check if we need to config + add short description

# FIXME: fails on build
# image optim
# optimizes PNG, JPEG, GIF and SVG file formats
# https://github.com/chrisdwheatley/netlify-plugin-image-optim
# [[plugins]]
#     package = "netlify-plugin-image-optim"

# subfont
# Subfont post-processes your web page to analyse you usage of web fonts,
# then reworks your webpage to use an optimal font loading strategy for the best performance.
# It will also aggressively subset your fonts to speed up the time to font download.
# https://github.com/Munter/netlify-plugin-subfont
[[plugins]]
    package = "netlify-plugin-subfont"

# inline critical css
# inline CSS above-the-fold content
# https://github.com/Tom-Bonnike/netlify-plugin-inline-critical-css
[[plugins]]
    package = "netlify-plugin-inline-critical-css"


# inline source
# inlines and compresses tags that use the inline attribute and supports <script>, <link>, and <img> tags
# https://github.com/Tom-Bonnike/netlify-plugin-inline-source
[[plugins]]
    package = "netlify-plugin-inline-source"

# FIXME: fails on build
# Config for the Netlify Build Plugin: netlify-plugin-minify-html
# [[plugins]]
#     package = "netlify-plugin-minify-html"

#     [plugins.inputs]
#     contexts = [ 'production' ]

#     [plugins.inputs.minifierOptions]
#         removeComments = true
#         collapseInlineTagWhitespace = true


# no more 404
# checks for broken INTERNAL links
# https://github.com/swyxio/netlify-plugin-no-more-404
[[plugins]]
    package = "netlify-plugin-no-more-404"

# checklinks
# check for broken INTERNAL and EXTERNAL links
# https://github.com/munter/netlify-plugin-checklinks
[[plugins]]
    package = "netlify-plugin-checklinks"

    [plugins.inputs]
        entryPoints = ["*.html"]
        recursive = true
        pretty = true
        checkExternal = true

# hashfiles
# Hashfiles sets you up with an optimal caching strategy for static sites,
# where static assets across pages are cached for as long as possible in the
# visitors browser and never have to be re-requested.
# https://github.com/Munter/netlify-plugin-hashfiles
[[plugins]]
    package = "netlify-plugin-hashfiles"

        [plugins.inputs]
            # Directory where content-addressable files are moved and immutable cache-headers are set
            staticDir = "/static/"

# html-validate
# adds the ability to validate HTML generated by your build
# https://github.com/oliverroick/netlify-plugin-html-validate
[[plugins]]
    package = "netlify-plugin-html-validate"

    [plugins.inputs]
        ext = "html"
        config = ".htmlvalidate.json"
# lighthouse
# runs a Lighthouse report on your site
# https://github.com/netlify/netlify-plugin-lighthouse
[[plugins]]
    package = "@netlify/plugin-lighthouse"

    # Set the plugin to run prior to deploy, failing the build if minimum thresholds aren't set 
    [plugins.inputs]
        # TODO: reactivate
        fail_deploy_on_score_thresholds = "false"

        # Set minimum thresholds for each report area
        [plugins.inputs.thresholds]
            performance = 0.98
            accessibility = 1
            best-practices = 1
            seo = 0.8 # Netlify previews are not indexed by search engines, so this needs to be lower

# a11y
# Check for accessibility issues on critical pages of your Netlify website.
# https://github.com/netlify-labs/netlify-plugin-a11y
[[plugins]]
    package = "netlify-plugin-a11y"

    [plugins.inputs]
        # TODO: reactivate
        failWithIssues = false

# debug cache
# Debug & verify the contents of your Netlify build cache
# https://github.com/netlify-labs/netlify-plugin-debug-cache
[[plugins]]
    package = "netlify-plugin-debug-cache"

# TODO: make sure this works!
# sitemap
# Generate a sitemap for your Netlify site
# https://github.com/netlify-labs/netlify-plugin-sitemap
[[plugins]]
    package = "@netlify/plugin-sitemap"

    [plugins.inputs]
        buildDir = "/"

# submit sitemap
# Submit your sitemap to search engines (Google, Yandex)
# https://github.com/cdeleeuwe/netlify-plugin-submit-sitemap
[[plugins]]
    package = "netlify-plugin-submit-sitemap"

    [plugins.inputs]
        # Time in seconds to not submit the sitemap after successful submission
        ignorePeriod = 600 # 10 minutes
        sitemapPath = "/sitemap.xml"
```
